integrations/Matomo/util.js

```diff
@@ -0,0 +1,295 @@
+import get from "get-value";
+import each from "@ndhoule/each";
+import logger from "../../utils/logUtil";
+import { getHashFromArray } from "../utils/commonUtils";
+
+/** If any event name matches with the goals list provided by the dashboard
+ * we will call the trackGoal with the id provided in the mapping.
+ */
+const goalIdMapping = (event, goalListTo) => {
+  each((val, key) => {
```

Comment:
Unable to open documentation for `each` where can i find its reference?

Reply:
It is here workspace/rudder-sdk-js/node_modules/@ndhoule/each/index.js

Reply:
Can't access that its fine will check this later